packages/client/components/PokerEstimateHeaderCard.tsx

```diff
@@ -44,7 +54,17 @@ export default createFragmentContainer(PokerEstimateHeaderCard, {
         integration {
           ... on JiraIssue {
             __typename
-            ...PokerEstimateHeaderCardJira_issue
+            issueKey
+            summary
+            descriptionHTML
+            jiraUrl: url
+          }
+          ... on JiraServerIssue {
```

Comment:
+1 I see this duplication happens twice in this PR. In the future we'll probably want to make an interface that handles both.

e.g.
```ts
JiraCloudIssue implements JiraIssue
JiraServerIssue implements JiraIssue
```

then, we could still pass `issue` into the component

Reply:
Yeah, I thought about it. We can do it in some next PRs